android-components/components/service/nimbus/src/main/java/mozilla/components/service/nimbus/messaging/NimbusMessagingStorage.kt

```diff
@@ -208,8 +206,7 @@ class NimbusMessagingStorage(
         }
     }
 
-    @VisibleForTesting
-    internal fun isMessageEligible(
+    fun isMessageEligible(
```

Comment:
Are we going to use this function outside of the `NimbusMessagingStorage`? If so we should have some comments documenting the function and its parameters :)

Reply:
Actually I only turned this method visible outside of tests because the test that uses it is now in another module. 

I [filed a bug to improve that](https://bugzilla.mozilla.org/show_bug.cgi?id=1823472) but for now I couldn't think of a cleaner way to solve the problem. 

In this case, would the best solution for now  be to document the function and its parameters? 

Reply:
Let's leave it as it is, this a good enough workaround until we move the tests to AC :)

Reply:
I updated the description of the bug to take this in account